### PR TITLE
Ignore non-existent tablet in clear_transaction_task()

### PR DESCRIPTION
**Pull request by lingbin**
#2300 

This commit also removes some duplicated logs, which are duplicated
printed inside and outside the function

### CODE REVIEW

**Comment by imay** [comment]
@lingbin 
Better to add an issue to record what is problem this patch intents to fix. Is it a bug or just an enhencement?

**Comment by lingbin** [comment]
> @lingbin
> Better to add an issue to record what is problem this patch intents to fix. Is it a bug or just an enhencement?

I add an issue #2300

File: be/src/olap/storage_engine.cpp [hunk 1]
```diff
@@ -495,10 +491,12 @@ void StorageEngine::clear_transaction_task(const TTransactionId transaction_id,
         // each tablet
         for (auto& tablet_info : tablet_infos) {
             // should use tablet uid to ensure clean txn correctly
-            TabletSharedPtr tablet = _tablet_manager->get_tablet(tablet_info.first.tablet_id, 
+            TabletSharedPtr tablet = _tablet_manager->get_tablet(tablet_info.first.tablet_id,
                 tablet_info.first.schema_hash, tablet_info.first.tablet_uid);
+            // The tablet may be dropped or altered, leave a INFO log and go on process other tablet
             if (tablet == nullptr) {
-                return;
+                LOG(INFO) << "tablet is no longer exist. tablet_id=" << tablet_info.first.tablet_id;
```
Comment (imay) [inline be/src/olap/storage_engine.cpp]: LOG schema hash and tablet_uid to be more clear

Comment (lingbin) [reply be/src/olap/storage_engine.cpp]: Done.

**Review by chaoyli — APPROVED** [review]

**Pull request merged**